recipes/llm/conda_build_config.yaml

```diff
@@ -0,0 +1,9 @@
+# see https://conda-forge.org/docs/maintainer/knowledge_base.html?#noarch-python
```

Comment:
Sorry, I directed you to the wrong way for doing this. Please rename this file to `conda-forge.yml` and add:

```yaml
noarch_platforms:
  - win_64
  - linux_64
  - osx_64
```

See https://conda-forge.org/docs/maintainer/knowledge_base.html#id6

Reply:
@ocefpaf thank you !